src/Template/Aefis/xml/e2b.ctp

```diff
@@ -270,7 +274,7 @@
                     if ($aefiListOfVaccine['suspected_drug']) echo 1 ;
                     else echo 2;
                 ?></drugcharacterization>
-                <medicinalproduct><?php echo $aefiListOfVaccine['vaccine_name']; ?></medicinalproduct>
+                <medicinalproduct><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefiListOfVaccine['vaccine_name']); ?></medicinalproduct>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate